website/source/docs/guides/backup.html.md

```diff
@@ -0,0 +1,87 @@
+
+Consul provides the [snapshot](https://consul.io/docs/commands/snapshot.html) command which can be run using the CLI command or the API. The `snapshot` command saves the point-in-time snapshot of the state of the Consul servers which includes:
+
+* KV entries 
+* the service catalog
+* prepared queries
+* sessions 
+* ACLs
+
+With [Consul Enterprise](/docs/commands/snapshot/agent.html), the `snapshot agent` command runs periodically and writes to local or remote storage (such as Amazon S3).
+
+By default, all snapshots are taken using `consistent` mode where requests are forwarded to the leader which verifies that it is still in power before taking the snapshot. Snapshots will not be saved if the cluster is degraded or if no leader is available. To reduce the burden on the leader, it is possible to [run the snapshot](/docs/commands/snapshot/save.html) on any non-leader server using `stale` consistency mode.
+
+This spreads the load across nodes at the possible expense of losing full consistency guarantees. Typically this means that a very small number of recent writes may not be included. The omitted writes are typically limited to data written in the last `100ms` or less from the recovery point. This is usually suitable for disaster recovery. However, the system can’t guarantee how stale this may be if executed against a partitioned server.
+
+## Create Your First Backup
+
+The `snapshot save` command for backing up the cluster state has many configuration options. In a production environment, you will want to configure ACL tokens and client certificates for security. The configuration options also allow you to specify the datacenter and server to collect the backup data. Below are several examples. 
+ 
+First, we will run the basic snapshot command with the all the defaults, including `consistent` mode.
+
+```sh
+$ consul snapshot save backup.snap
+Saved and verified snapshot to index 1176
+```
+The backup will be saved locally in the directory where we ran the command. 
+
+You can view metadata about the backup with the `inspect` subcommand. 
+
+```sh
+$ consul snapshot inspect backup.snap
+ID           2-1182-1542056499724
+Size         4115
+Index        1182
+Term         2
+Version      1
+```
+
+To understand each field review the inspect [documentation](https://www.consul.io/docs/commands/snapshot/inspect.html). Notably, the `Version` field does not correspond to the version of the data. Rather it is the snapshot format version. 
+
+Next, let’s collect the cluster data from a non-leader by specifying stale mode.
+
+```sh
+$ consul snapshot save -stale backup.snap
+Saved and verified snapshot to index 2276
+```
+
+Once ACLs and agent certificates are configured, they can be passed in with the following flags.
+
+```sh
+$ consul snapshot save -stale -token=<ead6787-23thbd-666789> -ca-file=</path/to/file> backup.snap
```

Comment:
Can we recommend using ENV for the token please? Including secrets in command arg is a big security risk and in this example where we are implicitly showing "how to do it in your secured production env" that is especially true. I think it should look like this (not tested though):

```suggestion
$ export CONSUL_HTTP_TOKEN="fc91b2b8-3631-4af4-8844-55b5c5e33235"
$ consul snapshot save -stale -ca-file=</path/to/file> backup.snap
```

You can set the ca file via ENV too if you want but that has no direct security concern.

Reply:
As a small thought, the example token here isn't a valid UUID string as its the wrong format and contains non-hex chars `th`. I think for clarity we should either show tokens as place holder like `<your ACL token>` or an actual UUID to demonstrate the real format.

Reply:
I am happy to change it to an ENV. I tested on my ACL enabled cluster and it works! I will also update the token to say `< your ACL token>` to be consistent.